Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/symptoms/calendar/SubmissionSymptomCalendarViewModel.kt

```diff
@@ -89,7 +89,7 @@ class SubmissionSymptomCalendarViewModel @AssistedInject constructor(
             )
         }
         performSubmission {
-            launch {
+            appScope.launch {
```

Comment:
Could this also lead to problems elsewhere? vm scope is often used after navigation

Reply:
the navigation still performed from vm scope. the issue here is that reporting the flag was also done from vm scope and we navigate immediately which cancels vm scope as we leave the screen and there was no chance to update the flag